tests/utils/skipif.php

```diff
@@ -77,6 +77,17 @@ function skip_if_not_replica_set_or_mongos_with_replica_set()
     is_replica_set(URI) or is_mongos_with_replica_set(URI) or exit('skip topology is not a replica set or sharded cluster with replica set');
 }
 
+function skip_if_no_transactions()
```

Comment:
I see you didn't remove `skip_if_not_replica_set_or_mongos_with_replica_set()`. Is that because it's still relevant for the change stream and causal consistency tests? I assume so, as they also don't need to distinguish 4.2+ for sharded clusters and thus could not utilize _this_ function.

Reply:
That is correct - tests for causal consistency require a replica set, regardless of it being standalone or behind mongos. The reason for the separate transactions helper here is to account for different versions depending on whether we're running on mongos with replica sets or a replica set.